src/Tgstation.Server.Host/Program.cs

```diff
@@ -67,6 +70,34 @@ public static async Task<int> Main(string[] args)
 				args = listArgs.ToArray();
 			}
 
+			if (InteropServices.RuntimeInformation.IsOSPlatform(InteropServices.OSPlatform.Linux))
+			{
+				using var proc = new Process
```

Comment:
This is unnecessary. There should be a getuid function available in the `Syscall` class somewhere.

Reply:
And platform specific checks should be abstracted, implement it as an `IsRoot` property of `ISystemIdentity`.